src/pages/index.js

```diff
@@ -7,96 +7,208 @@ import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import PopupWithForm from "../components/PopupWithForm.js";
 import PopupWithImage from "../components/PopupWithImage.js";
+import PopupWithConfirm from "../components/PopupWithConfirm.js";
 import Section from "../components/Section.js";
 import UserInfo from "../components/UserInfo.js";
 import {
-  initialCards,
+  profileAvatarModalSelector,
+  profileEditModalSelector,
+  cardModalSelector,
+  deleteCardModalSelector,
   config,
   profileTitleSelector,
   profileDescriptionSelector,
-  profileEditModalSelector,
+  profileAvatarSelector,
   profileEditButton,
   profileDescriptionElement,
   profileTitleElement,
-  cardModalSelector,
   addNewCardButton,
-  cardList,
+  cardListElement,
   previewImageModal,
+  cardTemplateElement,
+  profileAvatarPeniclIcon,
 } from "../utils/constants.js";
+import API from "../components/API.js";
+import { data } from "autoprefixer";
 
 /* ---------------------------------------------------------------------------------------------- */
-/*                                         Form Validator                                         */
+/*                                         Class Instances                                        */
 /* ---------------------------------------------------------------------------------------------- */
-
 const editProfileFormValidator = new FormValidator(config, profileEditModalSelector);
 const addCardFormValidator = new FormValidator(config, cardModalSelector);
-editProfileFormValidator.enableValidation();
-addCardFormValidator.enableValidation();
+const avatarFromValidator = new FormValidator(config, profileAvatarModalSelector);
```

Comment:
**Could be improved**

You can **universally** create instances of validators for all forms in the project storing them inside one object `formValidators`.  And then you can take any validator using attribute `name` of the form where you need to disable/enable the submit button or remove errors.

```js
const formValidators = {}

// enable validation
const enableValidation = (config) => {
  const formList = Array.from(document.querySelectorAll(config.formSelector))
  formList.forEach((formElement) => {
    const validator = new FormValidator(config, formElement)
    // here you get the name of the form
    const formName = formElement.getAttribute('name')

   // here you store a validator by the `name` of the form
    formValidators[formName] = validator;
   validator.enableValidation();
  });
};

enableValidation(config);

```

And now you can use them for disabling buttons or clearing errors:

```js

formValidators[ profileForm.getAttribute('name') ].resetValidation()

// or you can use a string – the name of the form (you know it from `index.html`)

formValidators['profile-form'].resetValidation()

```